subiquity/tests/api/test_api.py

```diff
@@ -609,14 +613,15 @@ async def test_basic_core_boot(self):
             await inst.post('/storage/v2/guided', data)
             v2resp = await inst.get('/storage/v2')
             [d] = v2resp['disks']
-            [p1, p2, p3, p4, p5] = d['partitions']
-            e1 = dict(offset=1 << 20, size=1 << 20, mount=None)
+            pgs = d['partitions']
+            [p4] = match(pgs, number=4)
+            # FIXME that gap probably isn't actually supposed to be there
+            [p1, g1, p2, p3, p4] = d['partitions']
```

Comment:
Or this one. Is this a comment about the gadget in the sample data or ...?

Reply:
Updated comment in https://github.com/canonical/subiquity/pull/1674